cardano-tx-generator/test/Main.hs

```diff
@@ -1,106 +0,0 @@
-{-# LANGUAGE QuasiQuotes #-}
-module Main (main) where
-
-import           Prelude
-import           Data.Maybe
-import           Test.Tasty
-import           Test.Tasty.HUnit
-import           Text.Heredoc
-import           Options.Applicative
-  
-import           Cardano.Benchmarking.Run (parseCommand)
-import           Cardano.Benchmarking.GeneratorTx.SizedMetadata
-
-
---import           Cardano.Benchmarking.MockServer as MockServer
-
-main :: IO ()
-main = defaultMain tests
-
-tests :: TestTree
-tests =  testGroup "cardano-tx-generator"
-  [
-    cliArgs
-  , sizedMetadata
-  , mockServer
-  ]
-  
-mockServer = testGroup "direct/pure client-server connect"
-  [ testCase "tx-send == tx-received" $ assertBool "tx-send == tx-received" True -- TODO !
-  ]
-
-sizedMetadata = testGroup "properties of the CBOR encoding relevant for generating sized metadat"
-  [ testCase "Shelley metadata map costs" $ assertBool "metadata map costs" prop_mapCostsShelley
-  , testCase "Shelley metadata ByteString costs" $ assertBool "metadata ByteString costs" prop_bsCostsShelley
-  , testCase "Allegra metadata map costs" $ assertBool "metadata map costs" prop_mapCostsAllegra
-  , testCase "Allegra metadata ByteString costs" $ assertBool "metadata ByteString costs" prop_bsCostsAllegra
-  , testCase "Mary metadata map costs"    $ assertBool "metadata map costs" prop_mapCostsMary
-  , testCase "Marymetadata ByteString costs"    $ assertBool "metadata ByteString costs" prop_bsCostsMary
-  , testCase "Test mkMetadata" $ assertBool "" True --WIP
-  ]
```

Comment:
This is basically what you called calibration of the sized-payload function.
Since sized payloads rely on several assumptions about the opaque particularities of the CBOR encoding.
It makes sense to at least test those assumptions.

Reply:
Oh, wow, my apologies @MarcFontaine, I was really, _really_ too quick over here!

Thank you for catching this!